lttoolbox/compiler.cc

```diff
@@ -129,10 +129,25 @@ Compiler::parse(string const &file, UString const &dir)
   xmlCleanupParser();
 
 
-  // Minimize transducers
-  for(auto& it : sections)
-  {
-    it.second.minimize();
+  // Minimize transducers: For each section, call transducer.minimize() in
+  // its own thread. This is the major bottleneck of lt-comp and sections
+  // are completely independent transducers.
+  std::vector<std::thread> minimisations;
+  for(std::pair<const UString, Transducer>& it : sections)
+  {
+    if(jobs) {
+      minimisations.push_back(
+        std::thread([](Transducer &t) { t.minimize(); },
+                    std::ref(it.second)));
+    }
+    else {
+      it.second.minimize();
+    }
+  }
+  if (jobs) {
```

Comment:
Don't need this `if`, because `minimisations` will be empty so the loop is a no-op.

Reply:
right, leftovers from the old array thing